requirements.txt

```diff
@@ -1,73 +1,15 @@
-alembic==1.4.2
-aniso8601==3.0.0
-appdirs==1.4.3
 APScheduler==3.5.1
-asn1crypto==0.24.0
-attrs==19.3.0
-awsebcli==3.14.1
-bcrypt==3.1.7
-black==19.3b0
-blessed==1.15.0
 blinker==1.4
-botocore==1.10.48
-cached-property==1.4.3
-cement==2.8.2
-certifi==2018.4.16
-chardet==3.0.4
-click==6.7
-colorama==0.3.9
-coverage==4.5.1
-cryptography==2.8
-docker==3.4.0
-docker-compose==1.21.2
-docker-pycreds==0.3.0
-dockerpty==0.4.1
-docopt==0.6.2
-docutils==0.14
-entrypoints==0.3
+coverage==5.3
 Flask==1.0.2
-Flask-JWT-Extended==3.10.0
+Flask-JWT-Extended==3.24.1
 Flask-Mail==0.9.1
-Flask-Migrate==2.5.2
+Flask-Migrate==2.5.3
 flask-restx==0.2.0
-Flask-SQLAlchemy==2.3.2
-Flask-Testing==0.7.1
-future==0.16.0
+Flask-Testing==0.8.0
```

Comment:
Do we need this for running the application?

Reply:
No, we can move it to dev-requirements as this is used for testing purpose. Will make the changes soon.

Reply:
Given you'd also need to make changes to the installation steps to run tests, I'd say no don't do it, instead open an issue.

If you feel you are up for it, please go ahead.

Reply:
I will open an issue instead after this pull request got approved.

Reply:
Also, we can move coverage too later after making changes in the installation steps and all